app/controllers/users/approvals_controller.rb

```diff
@@ -3,10 +3,9 @@ class Users::ApprovalsController < ApplicationController
   before_action :correct_url_user?
 
   def new
+    @presenter = Users::ApprovalsPresenter.new(current_user, params[:approvable_type])
```

Comment:
I'd suggest to give presenters more valuable names. So we could know what entity we are dealing with.